dev/docker-compose.yaml

```
@@ -210,5 +210,23 @@ services:
     networks:
       kafka:
 
+  kong:
+    image: kong/kong:3.6
```

```suggestion
    image: kong/kong-gateway:latest
```

1. This repo only serves Kong EE.
2. Should we pin to `3.6`? By default, this repo represents its latest and maybe test against the latest Kong EE is better.

---

I just needed a TCP proxy that supports routing by SNI and TLS passthrough, so really it shouldn't matter for my test case. My only concern with using `kong-gateway:latest` is if something in Kong EE breaks that functionality then this test will begin to fail even if this library is still functioning correctly.